Fix doc on SORTABLE and UNF (#3387)
RediSearch/RediSearch: docs/docs/reference/Sorting.md

# Sorting by Indexed Fields

> As of RediSearch 0.15, it is possible to bypass the scoring function mechanism, and order search results by the value of different document attributes (fields) directly - even if the sorting field is not used by the query. For example, you can search for first name and sort by last name.

Reviewer: ```suggestion
As of RediSearch 0.15, you can bypass the scoring function mechanism and order search results by the value of different document attributes (fields) directly, even if the sorting field is not used by the query. For example, you can search for first name and sort by last name.
```

## Declaring Sortable Fields

> When creating the index with `FT.CREATE`, you can declare `TEXT`, `TAG`, `NUMERIC`, and `GEO` attributes to be `SORTABLE`. When an attribute is sortable, we can later decide to order the results by its values with relatively low latency (it can still be sorted by its values, but with not as good latency). For example, in the following schema:

Reviewer: ```suggestion
When creating an index with `FT.CREATE`, you can declare `TEXT`, `TAG`, `NUMERIC`, and `GEO` attributes as `SORTABLE`. When an attribute is sortable, you can later decide to order the results by its values with relatively low latency (the property is still sorted by its values, but with not as good latency). For example, in the following schema:
```
